src/components/MostPopular/ProductItem/ProductItem.js

```diff
@@ -86,6 +89,16 @@ class ProductItem extends Component {
     showMessage(REMOVED_FROM_WISHLIST_NOTIFICATION);
   }
 
+  handleImageError() {
+    const { errored } = this.state;
+    if (!errored) {
+      this.setState({
+        productFirstImage: 'src/assets/img/productListPage/placeholder-image.jpeg',
+        errored: true
```

Comment:
better name suggestions: [hasError, isError]

Reply:
about classes: we've discussed it with Mira and Chrystia and have agreement to keep it the way it is. it was our misunderstading.